### PR TITLE
Make job queue working outside of browser

### PR DESCRIPTION
**Pull request by orzechdev**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1259 When no browser is present (window === undefined), SDK attempts to restore jobs from localStorage](https://app.clickup.com/t/2549495/SALEOR-1259)

File: src/jobs/JobsManager.ts [hunk 1]
```diff
@@ -165,6 +165,9 @@ export class JobsManager {
     G extends keyof IQueuedJobs,
     J extends keyof IQueuedJobs[G]
   >(jobGroup: G, jobName: J, state: boolean) {
+    if (!localStorage) {
```
Comment (patrys) [inline src/jobs/JobsManager.ts:168]: Can we use a fully qualified name like `window?.localStorage`? It will make it easier to find those in code.

Comment (orzechdev) [reply src/jobs/JobsManager.ts]: Sure

Comment (orzechdev) [reply src/jobs/JobsManager.ts]: Changed

**Review by patrys — APPROVED** [review]

**Pull request merged**